AwsCryptographyPrimitives/src/AwsCryptographyPrimitivesOperations.dfy

```diff
@@ -10,6 +10,7 @@ include "AESEncryption.dfy"
 include "Digest.dfy"
 include "RSAEncryption.dfy"
 include "Signature.dfy"
+include "AesKdfCtr.dfy"
```

Comment:
Ah I meant to add a note about this to the PR description. Without this, that file was accidentally only in scope for the test compilation instead of source. Because of that the extra `CreateStreamSuccess/Failure` methods weren't getting translated when I needed them.